ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -150,4 +152,7 @@ cloud_storage_base_url={{cloud_storage_base_url}}
 cloud_storage_cname_url={{ cloud_storage_cname_url | default('') }}
 cloud_storage_dial_bucketname={{ cloud_storage_dial_bucketname | default('dial') }}
 cloud_storage_path_prefix_dial={{ cloudstorage_relative_path_prefix_dial | default('DIAL_STORAGE_BASE_PATH') }}
-cloud_store_base_path_placeholder={{ cloud_store_base_path_placeholder | default('CLOUD_BASE_PATH') }}
\ No newline at end of file
+cloud_store_base_path_placeholder={{ cloud_store_base_path_placeholder | default('CLOUD_BASE_PATH') }}
+
+## Added to support multi storage
+sunbird_content_cloud_storage_type={{cloud_service_provider}}
```

Comment:
@ddevadat is this variable used only for OCI?

Reply:
this is added for the cloud store sdk selection based on the storage type. It could be azure, aws,gcp,oci

Reply:
is this still applicable considering the change in design? Based on this, will there be if else conditions within the microservice source code checkin for csp name? @maheshkumargangula @AmiableAnil

Reply:
I think this was added because initially Build/Core/Lms was build from tag/release-5.0.0_RC2 as per the sunbird ed release5.1.0 build document.

In this tag, value for CONTENT_CLOUD_STORAGE_TYPE is derived from sunbird_content_cloud_storage_type

https://github.com/Sunbird-Lern/lms-service/blob/780f632602dd38c1a92251beb08e166311665558/course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java#L784C1-L784C96

Thats the reason, when we initially deployed, we added this variable in the deployment env

However in release-5.0.2, this got changed
https://github.com/Sunbird-Lern/lms-service/blob/c2703d50f920f83089bf891cbebb796919441a37/course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java#L785

Now , value for CONTENT_CLOUD_STORAGE_TYPE is derived from sunbird_cloud_service_provider.

This makes the below line redundant in sunbird_lms-service.env
sunbird_content_cloud_storage_type={{cloud_service_provider}}

I will be removing it